Example/PaymentSheet Example/PaymentSheetUITest/CustomerSheetUITest.swift

```diff
@@ -120,10 +120,6 @@ class CustomerSheetUITest: XCTestCase {
 
         removeFirstPaymentMethodInList()
 
-        let doneButton = app.staticTexts["Done"]
-        XCTAssertTrue(doneButton.waitForExistence(timeout: 60.0))
-        doneButton.tap()
-
```

Comment:
You don't need to tap "Done" anymore when there are no more saved payment methods to edit. CustomerSheet now automatically ends 'edit' mode if there are no more editable payment methods.

Reply:
In the case of `CustomerSheet.remove.card.w.cbc.still.mp4` - looks like after deleting 4242, the UI transitions to the CBC edit screen, but was expecting exiting edit mode?